pages/api/discovery/[orgId].ts

```diff
@@ -42,9 +42,9 @@ export async function handler(req: NextApiRequest, res: NextApiResponse) {
       setIntermediateSession(req, res, intermediate_session_token)
       clearSession(req, res)
       if(mfa_required != null && mfa_required.secondary_auth_initiated == "sms_otp") {
-        return res.redirect(307, `/smsmfa?sent=true&org_id=${organization.organization_id}&member_id=${member.member_id}`);
+        return res.redirect(307, `/${organization.organization_slug}/smsmfa?sent=true&org_id=${organization.organization_id}&member_id=${member.member_id}`);
       } else {
-        return res.redirect(307, `/smsmfa?sent=false&org_id=${organization.organization_id}&member_id=${member.member_id}`);
+        return res.redirect(307, `/${organization.organization_slug}/smsmfa?sent=false&org_id=${organization.organization_id}&member_id=${member.member_id}`);
```

Comment:
is it worth pulling this out into a helper?
```
return redirectToSMSMFA(res, organization, member)
```

Reply:
https://github.com/stytchauth/stytch-b2b-nextjs-example/pull/20/commits/9ef1a7368fdd456d0c6c5078da564782be60e7ac